portfolio/src/main/webapp/comments.js

```diff
@@ -12,25 +12,71 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/**
+ * An object to manage the query string of a URL
+ * @const {URLSearchParams}
+ */
+const GET_PARAMS = new URLSearchParams ({
+  quantity: '5',
+  dateOrder: 'true'
+});
+
 /**
  * Fetches comments from the server and adds it to the DOM.
  */
 function getComments() {
-  fetch('/comments').then(response => response.json()).then((comments) => {
-    const commentsContainer = document.getElementById('comments-container');
+  const url = '/comments?' + GET_PARAMS.toString();
```

Comment:
No problem here, but just a tip, ES6 let you to make something call String interpolation, it is very handy.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Oh, I didn't know about that! Thanks for the tip!